.vib/elasticsearch/vib-publish.json

```diff
@@ -35,15 +35,13 @@
           "action_id": "trivy",
           "params": {
             "threshold": "CRITICAL",
-            "vuln_type": [
-              "OS"
-            ]
+            "vuln_type": ["OS"]
           }
         },
         {
           "action_id": "health-check",
           "params": {
-            "endpoint": "lb-elasticsearch-kibana-http",
+            "endpoint": "lb-elasticsearch-tcp-rest-api",
             "app_protocol": "HTTP"
           }
         }
```

Comment:
The `verify` stage in both files is not identical. Please, make sure to include `goss` and `cypress` actions here too.

Reply:
Oops. Fixed.